product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/01_vpc_endpoint.mdx

```diff
@@ -76,23 +81,28 @@ Now that your endpoint service is created, you can connect it to the cluster VPC
 
 1. From the navigation pane on the left, select **Endpoints** under **Virtual Private Cloud**, and then select **Create endpoint**.
 
-1. Enter a suitable name for the endpoint service.
+1. On the Create endpoint page:
+
+   1. Enter a suitable name for the endpoint service.
 
-1. Under **Service category**, select **Other endpoint services**.
+   1. Under **Service category**, select **Other endpoint services**.
 
-1. Under **Service Name**, enter the name of the endpoint service that you created earlier (`com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc`). Verify whether you have successfully allowed access to the endpoint by selecting **Verify service**.
+   1. Under **Service Name**, enter the name of the endpoint service that you created earlier (`com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc`). Verify whether you have successfully allowed access to the endpoint by selecting **Verify service**.
 
 1. Under VPC, select the client's VPC in which to create the endpoint. 
 
 1. Under **Subnets**, select the subnets (availability zones) in which to create the endpoint network interfaces. Enable the endpoint in all availability zones used by your application.
+<img width="956" alt="vpc_jj subnet select" src="https://user-images.githubusercontent.com/90636096/184053380-1b3dd815-04a6-469f-b1ab-5d4772ca9270.png">
 
 1. Select **Create endpoint**.
 
 ## Step 3: Accept and test the connection
 
 1. In your AWS account connected to BigAnimal, select **VPCs** and then select **Endpoint services**. 
+<img width="646" alt="end_pointservices" src="https://user-images.githubusercontent.com/90636096/184053704-33a5ab76-f9a1-42d4-bdf1-52dc365a4aae.png">
```

Comment:
I'd delete these last two also. The second one also has the issue of the names shown in the screenshot don't match what is in the text.